modular_skyrat/modules/loadouts/loadout_items/loadout_datum_inhands.dm

```diff
@@ -20,6 +20,10 @@ GLOBAL_LIST_INIT(loadout_inhand_items, generate_loadout_items(/datum/loadout_ite
 	name = "Cane"
 	item_path = /obj/item/cane
 
+/datum/loadout_item/inhand/cane/white
+	name = "White Cane"
+	item_path = /obj/item/cane/white
```

Comment:
Does the white cane actually do anything?

Reply:
Yes, it allows them to examine stuff they aren't holding and examine stuff faster.